### PR TITLE
Paseo v1.2.6 runtime

### PR DESCRIPTION
**Pull request by raynaudoe**
- Updated relay code to v1.2.6
- Updated patching scripts

### CODE REVIEW

File: relay/paseo/src/lib.rs [hunk 1]
```diff
@@ -273,7 +277,11 @@ impl pallet_preimage::Config for Runtime {
 }
 
 parameter_types! {
-	pub EpochDuration: u64 = EPOCH_DURATION_IN_SLOTS as u64;
+	pub EpochDuration: u64 = prod_or_fast!(
+		EPOCH_DURATION_IN_SLOTS as u64,
+		2 * MINUTES as u64,
+		"PAS_EPOCH_DURATION"
+	);
```
Comment (raynaudoe) [inline relay/paseo/src/lib.rs:284]: Not sure if these values are the correct ones

File: relay/paseo/src/lib.rs [hunk 2]
```diff
@@ -1450,12 +1467,13 @@ impl paras_registrar::Config for Runtime {
...
 	type DataDepositPerByte = ParaDataByteDeposit;
-	type WeightInfo = weights::runtime_common_paras_registrar::WeightInfo<Runtime>;
+	type WeightInfo = weights::polkadot_runtime_common_paras_registrar::WeightInfo<Runtime>;
 }
 
 parameter_types! {
 	// 12 weeks = 3 months per lease period -> 8 lease periods ~ 2 years
-	pub LeasePeriod: BlockNumber = prod_or_fast!(1 * WEEKS, 1 * DAYS, "DOT_LEASE_PERIOD");
+	pub LeasePeriod: BlockNumber = prod_or_fast!(1 * WEEKS, 1 * WEEKS, "PAS_LEASE_PERIOD");
+	pub LeaseOffset: BlockNumber = prod_or_fast!(1 * DAYS, 0, "PAS_LEASE_OFFSET");
```
Comment (raynaudoe) [inline relay/paseo/src/lib.rs:1476]: Also these ones

File: relay/paseo/src/lib.rs (hunk 1 above)
Comment (al3mart) [reply relay/paseo/src/lib.rs:284]: No, we need to revert these changes to what we had previously

File: relay/paseo/src/lib.rs [hunk 3]
```diff
@@ -1450,12 +1467,13 @@ impl paras_registrar::Config for Runtime {
...
 	type ParaDeposit = ParaDeposit;
 	type DataDepositPerByte = ParaDataByteDeposit;
-	type WeightInfo = weights::runtime_common_paras_registrar::WeightInfo<Runtime>;
+	type WeightInfo = weights::polkadot_runtime_common_paras_registrar::WeightInfo<Runtime>;
 }
 
 parameter_types! {
 	// 12 weeks = 3 months per lease period -> 8 lease periods ~ 2 years
-	pub LeasePeriod: BlockNumber = prod_or_fast!(1 * WEEKS, 1 * DAYS, "DOT_LEASE_PERIOD");
+	pub LeasePeriod: BlockNumber = prod_or_fast!(1 * WEEKS, 1 * WEEKS, "PAS_LEASE_PERIOD");
```
Comment (al3mart) [inline relay/paseo/src/lib.rs:1475]:
```suggestion
	pub LeasePeriod: BlockNumber = prod_or_fast!(1 * WEEKS, 1 * DAYS, "PAS_LEASE_PERIOD");
```

File: relay/paseo/src/lib.rs (hunk 2 above)
Comment (al3mart) [reply relay/paseo/src/lib.rs:1476]: We can get rid of `LeaseOffset`

File: relay/paseo/src/lib.rs [hunk 4]
```diff
@@ -3191,7 +3210,7 @@ mod remote_tests {
 	async fn try_fast_unstake_all() {
 		sp_tracing::try_init_simple();
 		let transport: Transport =
-			var("WS").unwrap_or("wss://paseo.rpc.amforc.com:443".to_string()).into();
+			var("WS").unwrap_or("wss://rpc.paseo.io:443".to_string()).into();
```
Comment (al3mart) [inline relay/paseo/src/lib.rs]: I don't believe this rpc is publicly accessible ?

Comment (raynaudoe) [reply relay/paseo/src/lib.rs]: Fixed!

**Review by hbulgarini — CHANGES_REQUESTED** [review]

File: relay/paseo/constants/src/lib.rs [hunk 5]
```diff
@@ -72,7 +72,7 @@ pub mod fee {
 	/// The block saturation level. Fees will be updates based on this value.
 	pub const TARGET_BLOCK_FULLNESS: Perbill = Perbill::from_percent(25);
 
-	/// Cost of every transaction byte at Kusama relay chain.
```
[the remark this answers is not in the document]
Comment (al3mart) [reply relay/paseo/constants/src/lib.rs:75]: ping @raynaudoe

[the remark this answers is not in the document]
Comment (raynaudoe) [reply relay/paseo/constants/src/lib.rs:75]: on it. Might be a bad renaming

**Review by al3mart — APPROVED** [review]
`try-runtime` failing run can be disregarded.

**Pull request merged**